mesheryctl/pkg/utils/platform.go

```diff
@@ -761,7 +761,7 @@ func ForceCleanupCluster() error {
 			if err := replicaSetInterface.Delete(context.TODO(), replicaSet.GetName(), metav1.DeleteOptions{
 				PropagationPolicy: &deletePolicy,
 			}); err != nil {
-				return err
```

Comment:
we decided to remove replicaSet as we already delete deployments above. remember??

Reply:
Ahh yeah, totally forgot about that